docs/source/auto_examples/TC_WS_TimeSeries.py

```diff
@@ -64,6 +64,6 @@
 plt.savefig('tracker_ws_Barry_ufsv1.png')
 
 # %%
-# .. figure:: images/thumb/sphx_glr_tracker_ws_Barry_thumb.png
+# .. figure:: /auto_examples/images/thumb/sphx_glr_tracker_ws_Barry_thumb.png
```

Comment:
Absolute paths?

Reply:
changed back to the relative paths

Reply:
Please check the latest commit below. Can I modify earlier commits? I have to figure out how.